metals/src/main/scala/scala/meta/internal/builds/MillBuildTool.scala

```diff
@@ -57,10 +55,23 @@ case class MillBuildTool(userConfig: () => UserConfiguration)
 
   }
 
+  private def bloopImportArgs(millVersion: String) = {
+    val isImportSupported = SemVer.isCompatibleVersion(
+      "0.9.10",
+      millVersion
+    ) && (SemVer.isLaterVersion(millVersion, "0.10.0-M1") || SemVer
+      .isCompatibleVersion("0.10.0-M4", millVersion))
```

Comment:
If it's supported then we can just have:
```suggestion
    val isImportSupported = SemVer.isCompatibleVersion(
      "0.9.10",
      millVersion
    )
```
I think?

Reply:
This way we break the support from `0.10.0-M1` to `0.10.0-M3` since it was introduced in M4 and backported in `0.9.10`.
This might be ok since they are milestone versions, but I tried to not break anything.